pkg/debug/constants/consts.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+import "zotregistry.io/zot/pkg/api/constants"
+
+const (
+	DebugGQLPlaygroundEndpoint = constants.RoutePrefix + "/_zot/debug/graphql-playground"
```

Comment:
Please don't concatenate these 2 strings here. It will cause problems later when we need to set up a subrouter for `/v2`.

Reply:
Removed the constants.RoutePrefix